# Pendant to ROS std_msgs/String.msg but with a message header

# Standard ROS message header
Header header
# Simple string containing data
string data
